apis/core.oam.dev/common/types.go

```diff
@@ -357,13 +380,6 @@ type WorkflowStatus struct {
 	StartTime metav1.Time `json:"startTime,omitempty"`
 }
 
-// SubStepsStatus record the status of workflow steps.
-type SubStepsStatus struct {
-	StepIndex int                     `json:"stepIndex,omitempty"`
-	Mode      WorkflowMode            `json:"mode,omitempty"`
```

Comment:
why index and mode no need?

Reply:
Substeps only execute in dag mode because the main step execute in step by step mode

Reply:
fine